tls/s2n_server_cert.c

```diff
@@ -86,6 +86,9 @@ int s2n_server_cert_recv(struct s2n_connection *conn)
 
 int s2n_server_cert_send(struct s2n_connection *conn)
 {
+    if (conn->actual_protocol_version == S2N_TLS13) {
+        GUARD(s2n_stuffer_write_uint8(&conn->handshake.io, 0));
+    }
     GUARD(s2n_send_cert_chain(&conn->handshake.io, conn->handshake_params.our_chain_and_key->cert_chain));
```

Comment:
I can't tell from reading this code what it actually does :) Maybe either the 0 should have a meaningful variable name, or there should be a why comment.

Reply:
added a comment and also a link to RFC which talks about certificate request context.